ci-operator/config/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master.yaml

```diff
@@ -41,7 +41,7 @@ tests:
   openshift_installer:
     cluster_profile: gcp
 - as: e2e-aws-olm
-  commands: make e2e
+  commands: JUNIT_DIRECTORY=$(ARTIFACTS)/junit make e2e
```

Comment:
Where is `$(ARTIFACTS)` set?

Reply:
Prow makes it available. See [here](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables)